apps/design-system/src/subjects/views/connectors/connector-details.tsx

```diff
@@ -34,6 +35,17 @@ inputComponentFactory.registerComponent(new RadialInput())
 
 const ConnectorsDetailsPageWrapper = (): JSX.Element => (
   <ConnectorDetailsPage
+    searchQuery=""
+    toEntity={noop}
+    toScope={noop}
+    entities={mockConnectorRefList}
+    isConnectorReferencesLoading={false}
+    setIsConnectorRefSearchQuery={noop}
+    currentPage={1}
+    totalPages={1}
+    goToPage={noop}
+    apiError={''}
+    apiConnectorRefError={''}
```

Comment:
to keep it cleaner, drop the optional arguments

Reply:
done